.github/workflows/ci.yaml

```
@@ -76,22 +76,24 @@ jobs:
           fi
       - name: SonarQube Scan
-        if: github.actor != 'dependabot[bot]' && env.SONAR_TOKEN != '' && steps.scoverage.outputs.report_path != ''
+        if: github.actor != 'dependabot[bot]' && env.SONAR_TOKEN != '' && steps.scoverage.outputs.report_paths != ''
         continue-on-error: true
         uses: SonarSource/sonarqube-scan-action@v4
         env:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         with:
-          args: -Dsonar.scala.coverage.reportPaths=${{ steps.scoverage.outputs.report_path }}
+          args: -Dsonar.scala.coverage.reportPaths=${{ steps.scoverage.outputs.report_paths }}
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)